packages/substyle/src/createSubstyle.js

```diff
@@ -68,20 +67,20 @@ function createSubstyle(
             'Optional second parameter must be a plain object.'
           )
 
-          const modifierKeys = filter(isModifier, selectedKeys)
-          const elementKeys = filter(isElement, selectedKeys)
+          const modifierKeys = selectedKeys.filter(isModifier)
+          const elementKeys = selectedKeys.filter(isElement)
 
           const collectElementStyles =
             elementKeys.length > 0
               ? (fromStyle: Object) =>
                   values(pickNestedStyles(fromStyle, elementKeys))
               : (fromStyle: Object) => [fromStyle]
 
-          const collectSelectedStyles = compose(
-            collectElementStyles,
-            (fromStyle: Object) =>
+          const collectSelectedStyles = (fromStyle: Object) => {
```

Comment:
Your PR currently adds 2 flow error, which can be easily fixed by changing this line:

```suggestion
          const collectSelectedStyles = (fromStyle: Object = {}) => {
```